src/entities/MovieTitle/MovieTitle.jsx

```diff
@@ -0,0 +1,47 @@
+import "./MovieTitle.css";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faEllipsisVertical } from "@fortawesome/free-solid-svg-icons";
+import { useState } from "react";
+
+export const MovieTitle = ({
+  imgUrl,
+  movieName,
+  releaseYear,
+  genres,
+  onClickCB,
+}) => {
+  const [isContextDropdownOpen, setContextDropdownOpen] = useState(false);
+
+  const handleToggleDropdown = () => {
+    setContextDropdownOpen((prevValue) => !prevValue);
+  };
+
+  return (
+    <div className="movie-title" onClick={onClickCB}>
+      <img src={imgUrl} alt="movie" className="movie-image" />
+      <div className="context-menu">
+        <FontAwesomeIcon
+          icon={faEllipsisVertical}
+          className="context-icon"
+          onClick={handleToggleDropdown}
+        />
+        <div className={`items ${isContextDropdownOpen ? "active" : "hidden"}`}>
```

Comment:
I believe we can use classNames library for this case

Reply:
done